packages/site/src/lib/components/modals/Contact.svelte

```diff
@@ -6,6 +6,10 @@
   import { createEventDispatcher } from 'svelte';
   import { apiFetch } from '$lib/client/apiFetch';
   import type { SupportRequestBody } from '../../../routes/api/email/support/+server';
+  import type { Address } from '../../../routes/api/email/send/mail-channels.interface';
+  import { dictionary } from '$lib/stores';
+
+  export let toManagers: Address[] = [];
```

Comment:
@Danble please rename to `additionalRecipients` as the contact modal should not be responsible for contributors page knowledge. It should only know about what lies inside itself. We're aiming for a descriptive title that can be used anywhere, in this case by the contributors page to add managers as additional recipients.